yarn.lock

```diff
@@ -2819,7 +2819,7 @@
     minimist "^1.2.0"
 
 "@cockroachlabs/ui-components@link:packages/ui-components":
-  version "0.2.24-alpha.3"
+  version "0.2.24-alpha.4"
```

Comment:
nit: This file could be excluded from this PR since this diff is from previous changes that were reverted.